pagopa/io-app
# feat(IT Wallet): [SIW-667] Add details screen for an issued credential

## Short description
This PR adds a detail screen to show  an issued credential info.

## List of changes proposed in this pull request
- Add `ItwCredentialDetailsScreen`

## How to test
Activate the wallet then try to add a new credential from the wallet home screen. After issuing from the wallet home screen try a single tap on the credential card image to show related details. Of course the longpress on the credential card image should work to try a presentation flow.
